src/components/common/accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classnames from 'classnames';
+import { AccordionProps, BodyProps, HeaderProps } from './types';
+
+const Header: React.FC<HeaderProps> = ({ children, ...props }) => {
```

Comment:
Each component should be in separate folder and follow such structure:

common/accordion/
  components/
    accordion-header/
      accordion-header.ts
    components.ts
  /accordion.tsx
  
Also avoid such style: 
```
Accordion.Body = Body;
Accordion.Header = Header;
```
There should be only one Accordion component, and you should pass all needed config to it and use Body and Header inside it